lib/model/QubitMenu.php

```diff
@@ -48,7 +48,8 @@ public function __toString()
     /**
      * Wrapper for BaseMenu::getPath() call to allow additional functionality
      *  option 'resolveAlias' - resolve aliases into full path
-     *  option 'getUrl' - resolve path to internal or external URL.
+     *  option 'getUrl' - resolve path to internal or external URL
+     *  option 'removeIndex' - trim index.php from the start of path name.
```

Comment:
Point at the end or not :P? I think it should be the same for all lines.

Reply:
Ah, good catch. I'd gotten a CS fixer error for a missing period at the end which is how I'd ended up here, but I've updated this now! There doesn't seem to be a convention for the options array (we don't seem to be documenting this elsewhere) but to me it makes sense to not have the period at the end for these since these are like params. This should be fixed now!